### ISSUE TITLE
Unipi 1.1 support

### ISSUE THREAD

**Issue by MaartenBremer** [mh-superbox/unipi-control#8]
What a wonderful project!

Would you be able to support the Unipi 1.1 as well? 

I tried running your project, but I am getting an error because /sys/bus/i2c/devices/1-0057/eeprom seems to be read when it is empty with the Unipi 1.1. 

`UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte`

**Comment by mh-superbox** [comment]
I don't have Unipi 1.1 but we can try if we can get it to work ;-)

Do you have this file? /sys/bus/i2c/devices/1-0050/eeprom

**Comment by MaartenBremer** [comment]
That would be great! 

I do have that file indeed:
`00000000  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
*
000000e0  fa 55 01 01 00 00 70 64  02 fd 00 00 00 00 00 00  |.U....pd........|
000000f0  40 b2 2d 63 40 b2 29 57  00 00 00 00 00 00 00 00  |@.-c@.)W........|
00000100`

**Comment by mh-superbox** [comment]
I did some research and found out that the **unipi 1.1 does not support modbus**. The query of the relays works via gpio.

But my code is designed for modbus. I think it's possible to extend it, but without hardware it's impossible for me.

In the evok code it works over the gpiobus: https://github.com/UniPiTechnology/evok/blob/master/evok/config.py#L274

I have modified the code in the develop branch so that `/sys/bus/i2c/devices/1-0050/eeprom` is read but this will not get the unipi 1.1 to work without other code modifications.

**Comment by MaartenBremer** [comment]
Thanks for your research! 

I actually do think that the UniPi 1.1 supports the TCP modbus is supported. Not only does [UniPi says so itself](https://forum.unipi.technology/topic/536/unipi-1-1-and-modbus-tcp), but the unipi-tools with inside the unipi_tcp_server (which provides the modbus over tcp functionality), seems to have code to support the UniPi 1.1.

On my own Pi it seems to detect the board without issue:
```
# /opt/unipi/tools/unipi_tcp_server -l 127.0.0.1 -p 502 -vvv
Running on unipi_model UNIPI1
UniPi TCP Modbus Server: Listening Connection Established RET:3
poll timeout = -1[ms]
Starting primary loop
```
but I do not know how to verify it is able to do actually do something.

**Comment by mh-superbox** [comment]
Can you start the unipi-control and work my modifications?
I have a modbus register yaml file for my device (https://github.com/mh-superbox/unipi-control/blob/main/src/unipi_control/installer/etc/unipi/hardware/neuron/L203.yaml)

When the Unipi 1.1 support modbus and you have a yaml file for your device than maybe it works ;-)
I have only found a modbus register map for unipi neuron devices (https://kb.unipi.technology/en:files:products:02-neuron)

I think the next problem is in this line: https://github.com/mh-superbox/unipi-control/blob/develop/src/unipi_control/config.py#L280

**Comment by XMaarten** [comment]
Yes, you are right about where it fails. Unfortunately I do not have that yaml file and I don't know how to create it.

**Comment by MaartenBremer** [comment]
Thanks for your help. The problem is indeed on that line now. 

I asked UniPi support if they have this file for the UniPi 1.1 card available as well.
